packages/wallet-specs/src/protocols/virtual-fund-as-leaf/protocol.ts

```diff
@@ -80,17 +79,17 @@ const createGuarantorChannel = {
   },
 };
 
-export function fundGuarantorArgs({
-  guarantorChannel,
-  ledgerId,
-  balances,
-}: ChannelsKnown): SupportStateArgs {
-  const amount = total(balances);
-  return {
-    channelId: ledgerId,
-    outcome: ethAllocationOutcome([{ destination: getChannelId(guarantorChannel), amount }]),
-  };
-}
+// function fundGuarantorArgs({
```

Comment:
Can this go?

Reply:
Why not.

It was commented out because
- it returns the wrong type, since `SupportState.Init` has changed
- it isn't used, but it probably will be soon.